script.js

```diff
@@ -11,3 +51,139 @@ function hideColumnContents(className) {
     }
   }
 }
+
+function adjustWidthOfColumns(firstColumnToBeAdjusted, secondColumnToBeAdjusted, headerOfFirstColumnToBeAdjusted, headerOfSecondColumnToBeAdjusted, firstColumnToBeAdjustedWidth, secondColumnToBeAdjustedWidth) {
+  for (const element of firstColumnToBeAdjusted) {
+    element.style.width = firstColumnToBeAdjustedWidth + 'px';
+    
+  }
+
+  headerOfFirstColumnToBeAdjusted.style.width = firstColumnToBeAdjustedWidth + 'px';
+
+  for (const element of secondColumnToBeAdjusted) {
+    element.style.width = secondColumnToBeAdjustedWidth + 'px';
+    
+  }
+
+  headerOfSecondColumnToBeAdjusted.style.width = secondColumnToBeAdjustedWidth + 'px';
+}
+
+//function determineMouseoverColumnline(columnlineTwo, columnlineThree, columnlineFour, table) {
+  // document.addEventListener('mousemove', function(event) {
+  //   const mouseIsOverColumnlineTwo = event.clientX == columnlineTwo.offsetLeft + 5 || event.clientX == columnlineTwo.offsetLeft - 5
+  //                                    && event.clientY >= table.offsetTop && event.clientY <= table.offsetTop + table.offsetHeight;
+                                     
+  //   const mouseIsOverColumnlineThree = event.clientX == columnlineThree.offsetLeft + 5 || event.clientX == columnlineThree.offsetLeft - 5;
+  //   const mouseIsOverColumnlineFour = event.clientX == columnlineFour.offsetLeft;
+
+  //   if (mouseIsOverColumnlineTwo) {
+  //     return columnlineTwo;
+  //   }
+
+  //   if (mouseIsOverColumnlineThree) {
+  //     return columnlineThree;
+  //   }
+
+  //   if (mouseIsOverColumnlineFour) {
+  //     return columnlineFour;
+  //   }
+  // });
+//}
+
```

Comment:
if it's not needed anymore it should be deleted

Reply:
Sorry, I just keep it for the case I need it. I will delete it later if thats okay :)